sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -129,6 +129,7 @@ public class SystemSchema extends AbstractSchema
   private static final long IS_AVAILABLE_TRUE = 1L;
   private static final long IS_OVERSHADOWED_FALSE = 0L;
   private static final long IS_OVERSHADOWED_TRUE = 1L;
+  private static final long TARGET_NUM_REPLICAS_UNKNOWN = -1L;
```

Comment:
Per the javadoc on line 121, the variables here are booleans represented as longs, so I'd move this variable above the javadoc.